modules/virt-about-reclaiming-persistent-volumes.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-about-reclaiming-persistent-volumes_{context}"]
+
+= About reclaiming persistent volumes
+
+In past releases, you accomplished persistent volume reclamation by setting
+the Storage Class *reclaimPolicy* to *recycle*. However, Kubernetes no
```

Comment:
Please review the [syntax highlighting guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference). All of these non-gui elements should be enclosed in backticks.

Reply:
Done.